20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+There is nothing wrong with leaving the patch file you created in the root directory for a while though, it should not interfere with any new development as it should be automatically excluded from any diffs, and it's handy to keep it around in case you need to re-work the patch later (although you can of course re-download your submitted patch file from the ticket).
+
+## Keeping On Top Of Your Tickets
+Even though you've submitted the best patch in the history of WordPress Core development, it doesn't mean you can sit back and relax, for any number of reasons your patch might not get reviewed, tested and committed through to trunk. You should keep tabs on your tickets and every now and then ensure they still apply and refresh the patches if they haven't gone anywhere yet. You may also get notifications by email when something changes on the tickets you have worked on.
+
+### Email Notifications
+There's a good chance that you'll get a few notifications by email when things change on the tickets you've worked on, make sure you read these as there may be things you need to do in response, such as re-work your patch or reply to a comment to clarify something.
+
+The best kind of notification is a commit notification, or what I call a "props" email. These are awesome, it's great to see your username given some props when a ticket is closed as complete, take a moment and pat yourself on the back, you earned it!
+
+### [My Patches](https://core.trac.wordpress.org/my-patches)
+Use this Trac report now and then to review the status of tickets you've submitted patches to but for some reason they have not yet been accepted, reviewed, tested or committed through to trunk.
+
+### [Tickets I've Participated In](https://core.trac.wordpress.org/my-comments)
+This is a slightly different report to the [My Patches][] report in that it includes tickets you've commented on. Maybe you've been reviewing or testing a patch.
+
+### [Timeline](https://core.trac.wordpress.org/timeline)
+OK, this is more for self congratulating, or just finding that ticket you remember doing something on but it's now closed.
+
+![Timeline – WordPress Trac](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Timeline-–-WordPress-Trac.png)
+
+By default this report shows all ticket events you've been involved in up to 90 days back. If you want to look a little further back, there is a little control panel in the top right you can use to set the start date further back or you can use the "Previous Period" and "Next Period" links to step backwards and forwards through your timeline.
+
+## Get Involved!
+Since we've started doing our WP Core Contrib Days at Delicious Brains we've enjoyed sharing what we've worked on with each, our successes and failures, and genuinely found it to be a very rewarding exercise, with each little contribution helping to nudge WordPress development forward just a tiny bit.
```

Comment:
> Since we've started doing our WP Core Contrib Days at Delicious Brains we've enjoyed sharing what we've worked on with each, our successes and failures

could do with a small rework

> Since we've started doing our WP Core Contrib Days at Delicious Brains we've enjoyed sharing after each day what we've worked on, our successes and failures